routes/auth.js

```diff
@@ -0,0 +1,86 @@
+import { Router } from "express";
+import bcryptjs from "bcryptjs";
```

Comment:
# Use bcrypt instead of bcryptjs
Bcrypt is **1.8 times** faster than bcryptjs in generating hash passwords and **1.8 times** faster in comparing function.

Reply:
noted.